server/server.js

```diff
@@ -18,6 +18,9 @@ app.use(bodyParser.urlencoded({ extended: true }));
 app.get('/booking', async (req, res) => {
   console.log('inside /booking');
   let campId = parseInt(req.query.campId);
+  if (!campId) {
+    campId = 0;
+  }
   let name = '';
   let init = {};
   await axios.get('http://localhost:3003/overview', { params: { campId: campId } })
```

Comment:
hey i recognize this 👯

Reply:
Yep all my calls to your API work like a charm!